modules/sdk/gradle-plugins-workspace/src/main/java/com/liferay/gradle/plugins/workspace/task/CreateClientExtensionConfigTask.java

```diff
@@ -171,16 +177,25 @@ public Set<ClientExtension> getClientExtensions() {
 		return _clientExtensions;
 	}
 
+	@InputFiles
```

Comment:
What is the difference between `@InputFile` and `@InputFiles` annotation in this context?

Reply:
`@InputFiles` allows for an empty file to be returned. `@InputFile` has validation that requires a file to not be empty if one is returned, hence the helper method in `NpmInstallTask` that returned `null` instead of an empty file.

See https://docs.gradle.org/current/userguide/validation_problems.html#input_file_does_not_exist